src/Calculator/Main.kt

```diff
@@ -0,0 +1,47 @@
+package Calculator
+
+fun main() {
+    println("Enter the size of the array:")
+    val n = readLine()!!
+    var arr = emptyArray<Int>()
+    println("Enter the items")
+    for (i in 0..(n.toInt() - 1)) {
+        arr += readLine()!!.toInt()
+    }
+    println("Enter the action")
+    val action = readLine()!!
+    println(
+            "Your action it is:$action  " +
+                    "Your answer it is:${calculator(action, arr)}"
+    )
+}
+
+
+fun calculator(action: String, arr: Array<Int>): Float {
+    var result = 0f
+    when (action) {
+        "+" -> for (i in 0..(arr.size - 1)) {
```

Comment:
move each when statement to the constants. Avoid using magic symbols. For example: 
`private val PLUS_OPERATION = "+"`